pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pyDiscRip"
+version = "1.0"
+authors = [
+  { name="AkBKukU", email="author@example.com" },
+]
+description = "Automate ripping optical discs and extracting data"
+readme = "README.md"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Operating System :: POSIX :: Linux",
+]
+dependencies = [
+    "pyudev",
+    "python-libdiscid",
+    "musicbrainzngs",
+    "unidecode",
+    "ffmpeg"
```

Comment:
Should be [`ffmpeg-python`](https://pypi.org/project/ffmpeg-python/), according to the API reference.